apis/apis.json

```diff
@@ -6,13 +6,13 @@
     "serviceYaml": "cloudasset_v1.yaml",
     "productName": "Google Cloud Asset Inventory",
     "productUrl": "https://cloud.google.com/resource-manager/docs/cloud-asset-inventory/overview",
-    "version": "1.1.0",
+    "version": "2.0.0-alpha00",
     "type": "grpc",
     "description": "Recommended Google client library to access the Google Cloud Asset Inventory API (v1).",
     "dependencies": {
       "Google.Api.Gax.Grpc": "3.0.0-alpha00",
-      "Google.LongRunning": "1.1.0",
-      "Google.Cloud.Iam.V1": "1.2.0",
+      "Google.LongRunning": "project",
+      "Google.Cloud.Iam.V1": "project",
```

Comment:
I considered that - but there's a lot of back and forth because of the rule that GA libraries can't use the defaults. I'm starting to wonder whether it would be simpler to remove the defaulting entirely... what do you think?

Reply:
(Will merge this one for now.)

Reply:
Maybe add a bit in generateprojects that cleans up apis.json for non ga that currently match the default dependency versions? If we don't have the default it's going to be tedious to change dependency versions for all when we need to I think.

Reply:
Will think about that. But I want more and more of this to be automated in general... currently we don't have anything that modifies apis.json automatically, but we'll need to do that within ~12 months for fully automated releases. I think tooling that bumps "everything it can" to a new version might be the best way forward.